tests/cli/test_multisig_address.py

```diff
@@ -22,16 +22,16 @@ def test_generate_address(self):
         # Last element is always empty string
         output.pop()
 
-        self.assertEqual(len(output), pubkey_count * 10 + 11)
+        self.assertEqual(len(output), pubkey_count * 11 + 11)
```

Comment:
Why this has changed?

Reply:
This test is kind of weird because it depends on the specific number of lines from the output. Because there is a new log output, the number of lines changed.

Reply:
Ah, now I understand.